doc/api/path.md

```diff
@@ -68,7 +68,8 @@ changes:
 * Returns: {string}
 
 The `path.basename()` methods returns the last portion of a `path`, similar to
-the Unix `basename` command.
+the Unix `basename` command. Trailing `/` (and `\\` on Windows) characters are
+ignored.
```

Comment:
Just noticed that "path delimiters" is not the best wording as well, considering how `path.delimiter` is defined. How about

> Note: On Windows, both the forward slash (`/`) and backward slash (`\`) are accepted as path segment separators; however, if separators are to be added by the Windows-specific implementations of the `path` methods, only the backward slash (`\`) will be used.